.github/workflows/test_stable.yml

```diff
@@ -62,8 +62,6 @@ jobs:
         python setup.py egg_info
         req_txt="botorch.egg-info/requires.txt"
         min_torch_version=$(grep '\btorch[>=]=' ${req_txt} | sed 's/[^0-9.]//g')
-        # HACK around the fact that pytorch does not offer a mac binary for 1.13.1 for py3.11 - TODO: Remove when bumping torch to 2.0.1
```

Comment:
Thanks for catching this! Do you know if there's a reason it's specifying 2.0.1 rather than 2.0? I guess it's fine if this flow passes.